ironfish/src/memPool/memPool.test.ts

```diff
@@ -99,26 +99,38 @@ describe('MemPool', () => {
   describe('orderedTransactions', () => {
     const nodeTest = createNodeTest()
 
-    it('returns transactions from the node mempool sorted by fees', async () => {
+    it('returns transactions from the node mempool sorted by fee rate', async () => {
       const { node } = nodeTest
       const { wallet, memPool } = node
       const accountA = await useAccountFixture(wallet, 'accountA')
       const accountB = await useAccountFixture(wallet, 'accountB')
       const accountC = await useAccountFixture(wallet, 'accountC')
-      const { transaction: transactionA } = await useBlockWithTx(node, accountA, accountB)
-      const { transaction: transactionB } = await useBlockWithTx(node, accountB, accountC)
+      const { block, transaction: transactionA } = await useBlockWithTx(
+        node,
+        accountA,
+        accountB,
+      )
+      const transactionB = block.minersFee
       const { transaction: transactionC } = await useBlockWithTx(node, accountC, accountA)
 
-      jest.spyOn(transactionA, 'fee').mockImplementationOnce(() => BigInt(1))
+      const transactionASize = getTransactionSize(transactionA.serialize())
+      const transactionBSize = getTransactionSize(transactionB.serialize())
+      const transactionCSize = getTransactionSize(transactionC.serialize())
+
+      // Miner's fee transaction has a smaller size than the normal transaction fixture
+      Assert.isTrue(transactionBSize < transactionASize)
+      Assert.isEqual(transactionASize, transactionCSize)
```

Comment:
nit: you could add some assertions here with `getFeeRate`

Reply:
Yeah, I think we could get rid of the `Assert`s and add `expect`s for these after mocking the fee (it'll probably have to use `mockImplementation` rather than `mockImplementationOnce`)

Reply:
Got rid of `Assert`s and added `expect`s for `getFeeRate`.